cypress/integration/T3.js

```diff
@@ -0,0 +1,38 @@
+import TaskPage from '../objects/taskPage';
+import Common from '../objects/common';
+
+const taskPage = new TaskPage();
+const common = new Common();
+
+describe('Task creation', function() {
+
+    beforeEach(function () {
+        cy.loginAs('Admin');
+        common.visit('/Tasks');
+    })
+
+    it('Admin creates a new task', function () {
+        
+        common.clickCertainButton('button','Create Task');
+        const taskName = Date.now().toString(); 
+        taskPage.inputIntoASpecificField('taskDetailsForm.name', taskName);
+        taskPage.inputIntoASpecificField('taskDetailsForm.description', Date.now());
+        taskPage.clickBillToClientDropDown();
+        const selection = taskPage.generateRandomBillToClientSelection();
+        taskPage.selectRandomBillToClientOption(selection);
+
+        if (selection === 'Yes') {
+            common.clearField('taskDetailsForm.rate');
+            taskPage.inputIntoASpecificField('taskDetailsForm.rate', Math.floor(Math.random()*50));
+        }
```

Comment:
it is nice approach, but you are gambling here, and there is a chance that you miss a bug. If for example bill to client = yes does not work (have bug) you might run test case n times in a row and it would not show you that there is a bug, and your test would be kind of random